static/main.js

```diff
@@ -0,0 +1,87 @@
+$(document).ready(function () {
+  $('.add_entry').on('submit', function(event){
+      event.preventDefault();
+      add_post();
+    });
+});
+
+
+
+
+function add_post() {
+    var title = $('#title').val();
+    var text = $('#text').val();
+    $.ajax({
+      url: '/add',
+      type: 'POST',
+      dataType: 'json',
+      data: {'title': title, 'text': text},
+      success: success
+    });
+}
+
+function open_edit() {
```

Comment:
Where is this function called?  I can't find the call site in your change set here.